sgt-back-end/index.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable no-console */
+
+const pg = require('pg');
+const express = require('express');
+const app = express();
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:lfz@localhost/studentGradeTable'
+});
+
+app.get('/api/grades', (req, res, next) => {
+
+  const sql = `
+    select *
+    from "grades"
+  `;
+
+  db.query(sql)
+    .then(result => {
+      const grade = result.rows;
+      res.status(200).json(grade);
+    })
+    .catch(err => {
+      console.error(err);
+      res.status(500).json({
+        error: 'An unexpected error occurred.'
+      });
+    });
+});
+
+app.use(express.json());
```

Comment:
While this won't cause an issue in this project it is best to apply all your middleware before any of your endpoints.